std/experimental/allocator/building_blocks/region.d

```diff
@@ -830,7 +822,7 @@ version(Posix) struct SbrkRegion(uint minAlign = platformAlignment)
             || assert(0);
```

Comment:
I'm guessing you have missed `expand`s function attributes reordering

Reply:
Done.